darwin/client.py

```diff
@@ -455,7 +455,7 @@ def login(cls, email: str, password: str, projects_dir: Optional[Path] = None):
             refresh_token=data["refresh_token"],
             api_url=api_url,
             base_url=Client.default_base_url(),
-            projects_dir=projects_dir,
+            projects_dir=str(projects_dir),
```

Comment:
What would be broken if we still passed `project_dir` as `Path`?
Would it be possible to cast it to `str` where the issue occurs?